pkg/datarepair/repair.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package datarepair
+
+import (
+	"storj.io/storj/pkg/pb"
+)
+
+//RepairQueue is the interface for the data repair queue
+type RepairQueue interface {
+	Add(qi *pb.QueueItem) error
+	AddAll(qis []*pb.QueueItem) error
+	Remove(qi *pb.QueueItem) error
+	GetNext() pb.QueueItem
```

Comment:
You also need something for the case when the datarepair fails. When you get an item from queue, repairer crashes, it should still be handled properly.

Reply:
one thought here - the repair checker system is going to consistently continue to add at risk files to the queue. if we drop a job every so often (hopefully rarely), it will get added in the next repair checker run anyway so it's okay

Reply:
The question is how often will it detect the files at risk? But I can see some approaches that could have a close-to live view of the things needing repair.

Reply:
i think we should target at least one run of the repair checker every 24 hours until we get the reverse index built (will be much closer-to-live)